### PR TITLE
Feat/add function

### PR DESCRIPTION
**Pull request by xz5n15**

### CODE REVIEW

File: gui-prototype-1/popular/pop_ingredients.json [hunk 1]
```diff
@@ -0,0 +1,103 @@
+<<<<<<< HEAD
```
Comment (andreaseliasson) [inline gui-prototype-1/popular/pop_ingredients.json]: This line should be deleted

File: gui-prototype-1/popular/pop_ingredients.json [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+       "id": 9,
+       "name": "carrots"
+   },
+
+   {
+       "id": 10,
+       "name": "milk"
+   }
+]
+=======
```
Comment (andreaseliasson) [inline gui-prototype-1/popular/pop_ingredients.json]: Delete this line

File: gui-prototype-1/popular/pop_ingredients.json [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+       "id": 9,
+       "name": "carrots"
+   },
+
+   {
+       "id": 10,
+       "name": "milk"
+   }
+]
+>>>>>>> 5be1df666241c5deea7591c5a14348aaf37fb048
```
Comment (andreaseliasson) [inline gui-prototype-1/popular/pop_ingredients.json]: Delete this line

**Review by andreaseliasson — COMMENTED** [review]
How can we check the html and graphs?

File: gui-prototype-1/popular/pop_ingredients.json (hunk 3 above)
Comment (xz5n15) [reply gui-prototype-1/popular/pop_ingredients.json]: okay!

**Comment by xz5n15** [comment]
i did not combine it in the main html, you can directly run the html file in the pycharm application, and the app.py should run before the bubble html,  i will try to combine it in the main html.

**Pull request closed without merging**